goss/goss-windows.yaml

```diff
@@ -1,31 +1,163 @@
 command:
+  Powershell:
+    exec: powershell (Get-Host).Version
+    exit-status: 0
```

Comment:
Not blocking: this is the default expected behavior in POSIX (and in `goss` of course). 
Can be removed to simplify (not in this PR though)